modules/test/company-sample-data-generation-test/src/testIntegration/java/com/liferay/company/sample/data/generation/test/CompanySampleDataGenerationTest.java

```diff
@@ -78,7 +78,7 @@
 /**
  * @author Hai Yu
  */
-@DataGuard(scope = DataGuard.Scope.CLASS)
+@DataGuard(scope = DataGuard.Scope.NONE)
```

Comment:
@slnn We want this scope to be "CLASS" on CI. Please don't change it. When running it locally we can manually change it or automate the changing using other ways.